packages/components/src/components/hds/side-nav/base.ts

```diff
@@ -0,0 +1,19 @@
+import TemplateOnlyComponent from '@ember/component/template-only';
+
+export interface HdsSideNavBaseSignature {
+  Blocks: {
+    root?: [];
```

Comment:
@alex-ju looking at the `base.hbs` file, there's no difference between `root` and `header/body/footer`. Should this one too be `any`?

Reply:
in `base.hbs` yes, there's no difference – I forgot about it because when `SideNav::Base` is used in `index.hbs` it didn't have a hash (so there was no need to make it generic). We can make it `any`, for consistency.